src/extensions/replay/sessionrecording.ts

```diff
@@ -734,6 +741,17 @@ export class SessionRecording {
         }
     }
 
+    private _pageViewFallBack() {
+        if (this.instance.config.capture_pageview || !window) {
+            return
+        }
+        const currentUrl = this._maskUrl(window.location.href)
+        if (this._lastHref !== currentUrl) {
+            this._tryAddCustomEvent('$url_changed', { href: currentUrl })
```

Comment:
If we need that for mobile, in case people disable `captureScreenViews`, maybe we make an agnostic name here?
Web uses $pageview, Mobile uses $screen, not sure if there's something that would work for both.

Reply:
I'm happy to keep it, but we are adding more semantically identical things with different names so they can be easily unified. However, I agree that `href` won't make sense anyway so unifying is still suboptimal.